SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialNetDriverLoadBalancingHandler.cpp

```diff
@@ -29,44 +29,44 @@ void FSpatialNetDriverLoadBalancingContext::UpdateWithAdditionalActors()
 	}
 }
 
-EActorMigrationResult FSpatialNetDriverLoadBalancingContext::IsActorReadyForMigration(AActor* Actor)
+SpatialGDK::EActorMigrationResult FSpatialNetDriverLoadBalancingContext::IsActorReadyForMigration(AActor* Actor)
 {
 	if (!Actor->HasAuthority())
```

Comment:
Possibly add `using SpatialGDK::EActorMigrationResult` to save on code?

Reply:
possibly a good idea - done